packages/harmony/src/icons/IconDetails.stories.mdx

```diff
@@ -74,7 +83,7 @@ Locate the export panel in the bottom right corner of figma. (Export panel is on
 ⚠️ Verify the export settings are correct. (See screenshot) If not,
 go back to step 2.
 
-<img src={verifyIconSettings} />
+<img width={471} src={verifyIconSettings} />
```

Comment:
seems like an odd number. Is that just the size of the image?

Reply:
Yeah exactly, with the flex stuff, it was making them full width. If there is a better way lmk

Reply:
You could try `content-fit: none` on the image? idk if that's exactly the issue though